src/libs/actions/Report.js

```diff
@@ -902,7 +903,10 @@ function deleteReportComment(reportID, reportAction) {
  * @returns {Array}
  */
 const extractLinksInMarkdownComment = (comment) => {
-    const regex = /\[[^[\]]*\]\(([^()]*)\)/gm;
+    const regex = new RegExp(
+        `\\[(?:[^\\][]*(?:\\[[^\\][]*][^\\][]*)*)]\\(${URL_REGEX}\\)(?![^<]*(<\\/pre>|<\\/code>))`,
+        'gm',
+    );
```

Comment:
I should have mentioned this when reviewing your proposal, but I think it would actually be better to import the exact RegExp from expensify common so that we don't have any code duplication.

Already you can see that we have [changed this in expensify common](https://github.com/Expensify/expensify-common/blob/58503584940d5b112783d8d8115b63996599433c/lib/ExpensiMark.js#L109-L112), so if we import the exact RegExp from there then they will always be in sync. So we'll need to make a change in expensify-common too.

Reply:
So do I need to create a PR in Expensify-common as well?

Reply:
Let me explain. We uses match[1] to extract URLs here
https://github.com/Expensify/App/blob/2182d0cb6eee479fa1c4d6f3fd9dd9cc9e97adbf/src/libs/actions/Report.js#L908-L910

`?:` means non-capture group. If we don't use it, we may need to use match[2]

In order to sync with expensify-common as you mentioned above, I need to create a PR in expensify-common(need to add named export for link regex) and use match[2] without `?:`

Reply:
It looks like the latest changes of `expensify-common` isn't used in App yet.

Reply:
@aimane-chnaif </pre> doesn't work? How can I reproduce?